cmd/explaintest/t/new_character_set_invalid.test

```diff
@@ -15,3 +15,16 @@ insert into t values ('À', 'ø', '😂');
 insert into t values ('中文À中文', 'asdføfdsa', '字符集😂字符集');
 insert into t values (0x4040ffff, 0x4040ffff, 0x4040ffff);
 select * from t;
+
```

Comment:
I think `set @@sql_mode = default` is better

Reply:
Done.